conf/messages

```diff
@@ -2682,7 +2682,7 @@ claimToAdjustPoa.youCannotGoBack.bullet3Link             = homepage
 # opt out confirm
 optout.confirmOptOut.heading            = Confirm and opt out for the {0} to {1} tax year
 optout.confirmOptOut.desc               = If you opt out, you can submit your tax return through your HMRC online account or compatible software.
-optout.confirmOptOut.quarterly-updates  = You have <b>{0} quarterly updates submitted</b> for this tax year. If you continue, these updates will be deleted from our records. You’ll need to include any income from these updates in your tax return.
+optout.confirmOptOut.quarterly-updates  = You have {0} quarterly updates submitted for this tax year. If you continue, these updates will be deleted from our records. You’ll need to include any income from these updates in your tax return.
```

Comment:
Slight content difference from ticket in final sentence "_You will_ need to include..." instead of "You'll need to include..."
<img width="853" height="272" alt="Screenshot 2025-08-14 at 10 54 31" src="https://github.com/user-attachments/assets/f1f1f3b6-04f0-4c62-9f5d-24653a395734" />

Reply:
cheers Adam will update